pkgs/by-name/iv/ivm/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, makeWrapper
+, cargo
+, llvm_16
+, stdenv
+, libffi
+, libz
+, libxml2
+, ncurses
+, ...
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ivm";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "inko-lang";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-z0oo1JUZbX3iT8N9+14NcqUzalpARImcbtUiQYS4djA=";
+  };
+
+  cargoHash = "sha256-EP3fS4lAGOaXJXAM22ZCn4+9Ah8TM1+wvNerKCKByo0=";
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  fixupPhase = ''
+    runHook preFixup
+
+    wrapProgram $out/bin/ivm \
+      --prefix PATH : ${lib.makeBinPath [ cargo llvm_16.dev stdenv.cc ]} \
+      --prefix LIBRARY_PATH : ${lib.makeLibraryPath [
```

Comment:
```suggestion
      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [
```

? Not sure

Reply:
`LD_LIBRARY_PATH` doesn't work here, but `LIBRARY_PATH` probably isn't the best option either.
Afaik, `LD_LIBRARY_PATH` specifies where to find libraries at runtime, but the libraries provided here are needed at build-time (not the build-time of `ivm` itself, but the time when `ivm` builds `inko`).
I dislike using `LIBRARY_PATH` here because it seems to be a gcc specific variable, but I don't know how else to achieve this, and at least this way `ivm` works the way it's supposed to (i.e. `ivm install latest` downloads and builds the latest `inko` version, compiled with all the right dependencies and libraries).